app/src/main/java/com/jalasoft/routesapp/data/remote/managers/TourPointManager.kt

```diff
@@ -0,0 +1,24 @@
+package com.jalasoft.routesapp.data.remote.managers
+
+import android.content.Context
+import com.jalasoft.routesapp.data.model.remote.TourPoint
+import com.jalasoft.routesapp.data.model.remote.TourPointPath
+import com.jalasoft.routesapp.data.remote.interfaces.TourPointRepository
+import com.jalasoft.routesapp.util.PreferenceManager
+import com.jalasoft.routesapp.util.helpers.FirebaseCollections
+
+class TourPointManager(private val firebaseManager: FirebaseManager) : TourPointRepository {
+    override suspend fun getAllTourPoints(context: Context): List<TourPointPath> {
+        val currentCityId = PreferenceManager.getCurrentCityID(context)
+        val cityRef = firebaseManager.db.document("Cities/$currentCityId")
```

Comment:
Maybe use the enums we have and concat the id

Reply:
done